content/concepts/introduction/protocols/overview.md

```diff
@@ -1,206 +1,110 @@
 ---
-title: "Overview"
-description: There are protocols everywhere you look when you're writing network applications, and libp2p is especially thick with them.
```

Comment:
I like the zest of this earlier description, can we keep it?

Reply:
yes